tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -35,25 +37,31 @@ private static string ResolveManagedProfilerDirectory()
             var tracerHomeDirectory = ReadEnvironmentVariable("DD_DOTNET_TRACER_HOME") ?? string.Empty;
             var fullPath = Path.Combine(tracerHomeDirectory, tracerFrameworkDirectory);
 
+            if (!Directory.Exists(fullPath))
```

Comment:
Since the goal of this PR is to avoid exceptions that could crash the process, we should probably wrap `Directory.Exists()` in a `try/catch` in case it throws an exception. Possible causes could be an invalid value in `DD_DOTNET_TRACER_HOME` or the lack of ACL permissions.

Maybe even in a helper method like `ReadEnvironmentVariable()`, used a few lines above.

Reply:
I take this back. Apparently `Directory.Exists()` can _never_ throw.